.gitignore

```diff
@@ -0,0 +1 @@
+!icons/
```

Comment:
not sure why this would be needed

Reply:
My local repo didn't add the changed files?
I assume theres `Icon?` in my `.gitignore_global` to prevent OSX-filesystem-madness.